examples/collada.html

```diff
@@ -54,52 +55,37 @@
             // These will deform iTowns globe geometry to represent terrain elevation.
             promiseElevation.push(itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addLayerCb));
             promiseElevation.push(itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addLayerCb));
-
-            function loadCollada(url) {
-                var model;
-                // loading manager
-                var loadingManager = new itowns.THREE.LoadingManager(function _addModel() {
-                    globeView.scene.add(model);
-                    globeView.notifyChange();
-                });
-                // collada loader
-                var loader = new itowns.THREE.ColladaLoader(loadingManager);
+            
+            var promiseCollada = load('Collada', 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/models/collada/building.dae')
```

Comment:
- You should explain how to write the `format`. Because you can not write 'collada' or 'dae', because it's according to the file name of the loader. You could also link with [this page](https://github.com/mrdoob/three.js/tree/dev/examples/js/loaders).

- a another method :

```js
const supportedLoaders = new Map([
    ['collada', ColladaLoader],
    ['dae', ColladaLoader],
    ['fbx', FBXLoader],
    //  ...
]);
 const format = 'Dae';
 const getLoaderNameFile = supportedLoaders.get(format.toLowerCase());
if (!getLoaderNameFile) {
 // unsupported format 
}
```

- You could add the space name `load` to clarify where the loader comes from :
`ThreeLoader.load(format, url)` because the loader return THREE object

Reply:
ok for the namespace